l10n_ch_import_cresus/wizard/l10n_ch_import_cresus.py

```diff
@@ -0,0 +1,344 @@
+            inverted_default_value['date'] = None
+            inverted_default_value['ref'] = None
+            inverted_default_value['journal_id'] = None
+            inverted_default_value['period_id'] = None
+            if is_negative:
+                inverted_default_value['line_id/debit'] = abs(decimal_amount)
+                inverted_default_value['line_id/credit'] = 0.0
+                inverted_default_value['line_id/account_id']\
+                    = line_cresus['credit']
+            else:
+                inverted_default_value['line_id/debit'] = 0.0
+                inverted_default_value['line_id/credit'] = abs(decimal_amount)
+                inverted_default_value['line_id/account_id']\
+                    = line_cresus['credit']
+                # Search for account that have a deferal method
+            if tax_current or line_cresus['analytic_account']:
+                current_account_id = account_obj.search([
+                    ('code', '=',
+                     inverted_default_value['line_id/account_id'])])
+                if current_account_id:
+                    current_account = account_obj.browse(current_account_id)[0]
+                    if current_account.user_type.close_method == 'none':
+                        if tax_current:
+                            tax_code_inverted = tax_current['name']
+                    analytic_code_inverted = line_cresus['analytic_account']
+            inverted_default_value['line_id/account_tax_id']\
+                = tax_code_inverted
+            inverted_default_value['line_id/analytic_account_id']\
+                = analytic_code_inverted
+            new_openerp_data.append(inverted_default_value)
+
+        return new_openerp_data
+
+    def _load_data(self, data):
+        """Function that does the load of parsed CSV file.
+
+        If will log exception and susccess into the report fields.
+
+        :param data: CSV file content (list of data list)
+        """
+        # Change data from dict to list of array
+        data_array = []
+        for data_item_dict in data:
+            data_item = []
+            for item in self.HEAD_OPENERP:
+                data_item.append(data_item_dict[item])
+            data_array.append(data_item)
+        state = msg = None
+        try:
+            res = self.env['account.move'].load(self.HEAD_OPENERP,
+                                                 data_array)
+            state, msg = self._manage_load_results(res)
+        except Exception as exc:
+            ex_type, sys_exc, tb = sys.exc_info()
+            tb_msg = ''.join(traceback.format_tb(tb, 30))
+            _logger.error(tb_msg)
+            _logger.error(repr(exc))
+            msg = _("Unexpected exception.\n %s \n %s" % (repr(exc), tb_msg))
+            state = 'error'
+        finally:
+            if state == 'error':
+                self.env.cr.rollback()
+            self.write({'report': msg})
```

Comment:
As you have only one wizard, you can do a direct assignation

`self.report = msg`